core/editable.js

```diff
@@ -2081,6 +2082,17 @@
 			return nodesData;
 		}
 
+		// Prevent inserting div into div when Editor is in ENTER_DIV mode, and target element is an empty div.
+		function preventInsertingDiv( nodeName, element, editor ) {
```

Comment:
The second parameter can use some better name, e.g. `container`. Now `nodeName` and `element` seem to be about the same element.